tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py

```diff
@@ -82,9 +82,9 @@ def __init__(self, boardConfig):
         OtaAfrProject.APPLICATION_VERSION_PATH = self._buildProject + '/include/aws_application_version.h'
         OtaAfrProject.CLIENT_CREDENTIAL_KEYS_PATH = self._buildProject + '/include/aws_clientcredential_keys.h'
         OtaAfrProject.OTA_CODESIGNER_CERTIFICATE_PATH = 'demos/include/aws_ota_codesigner_certificate.h'
-        OtaAfrProject.OTA_BOOTLOADER_CONFIG_PATH = 'demos/ota/bootloader/utility/user-config/ota-descriptor.config'
-        OtaAfrProject.OTA_BOOTLOADER_CERTIFICATE_PATH = 'demos/ota/bootloader/utility/codesigner_cert_utility/aws_ota_codesigner_certificate.pem'
-        OtaAfrProject.OTA_FACTORY_IMAGE_GENERATOR_PATH = 'demos/ota/bootloader/utility/factory_image_generator.py'
+        OtaAfrProject.OTA_BOOTLOADER_CONFIG_PATH = 'vendors/microchip/boards/curiosity_pic32mzef/bootloader/bootloader/utility/user-config/ota-descriptor.config'
```

Comment:
I know this is supported only in microchip currently, but we might see this supported on other boards in future. I guess just to fix quickly for now this is fine.

Reply:
Need to add conditional clauses at many places for this. I will add it later today and clean it after we are done with todays testing :-)